tests/php/test-class-amp-youtube-embed-handler.php

```diff
@@ -140,8 +140,6 @@ public function test_video_override() {
 	}
 
 	public function get_conversion_data() {
-		$loading_attribute = version_compare( get_bloginfo( 'version' ), '5.5-alpha', '>' ) ? 'loading="lazy" ' : '';
```

Comment:
@westonruter Do you think we should specify a width and height for these image placeholders? Otherwise, WP (5.5 alpha at least) will bail when attempting to add the `loading="lazy"` attribute. Personally, I think since the `layout` is set to `fill` that would not be needed.

PS: This change was made as these tests are failing on WP `trunk`.

Reply:
Correct, since the `layout` is `fill` then no dimensions are needed.